custom_components/tahoma/number.py

```diff
@@ -28,6 +28,36 @@
         max_value=4,
         entity_category=EntityCategory.CONFIG,
     ),
+    OverkizNumberDescription(
+        key=OverkizState.CORE_ECO_ROOM_TEMPERATURE,
+        name="Eco temperature",
+        icon="mdi:content-save-cog",
```

Comment:
Should we change this to temperature? They are in the config section already, thus perhaps best to reflect what the setting does?

 ```suggestion
        icon="mdi:thermometer",
```

Or `mdi:home-thermometer-outline` for room temperature and `mdi:sun-thermometer-outline` for Freeze protection temperature.